lib/puppet/type/netapp_ldap_client.rb

```diff
@@ -69,6 +70,7 @@
 simple - Simple bind,
 sasl - Simple Authentication and Security Layer (SASL) bind"
     newvalues(:anonymous, :simple, :sasl)
+    defaultto(:simple)
```

Comment:
The copy/pasted docs mention various defaults under different situations. Either the code should match this, or the docs should be updated.